core/dbt/events/test_types.py

```diff
@@ -14,7 +14,7 @@
 
 @dataclass
 class IntegrationTestInfo(InfoLevel, NoFile):
-    msg: str
+    msg: str = ''
```

Comment:
I don't like having defaults we hope are never going to get used. But since it's just test events I can live with this if we really need it.

Reply:
I forgot to update the test_types file. Have done this now.